change/@fluentui-react-datepicker-compat-881179ec-abf2-4f4f-8a61-1b88fd0220e5.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "breaking: Update Datepicker to use react-calendar-compat and remove Calendar related exports.",
```

_nit:_ We normally specify breaking changes with all caps in change files

```suggestion
  "comment": "BREAKING: Update Datepicker to use react-calendar-compat and remove Calendar related exports.",
```